softvis3d-frontend/src/components/scene/SceneKeyInteractions.ts

```diff
@@ -0,0 +1,62 @@
+///
+/// softvis3d-frontend
+/// Copyright (C) 2016 Stefan Rinderle and Yvo Niedrich
+/// stefan@rinderle.info / yvo.niedrich@gmail.com
+///
+/// This program is free software; you can redistribute it and/or
+/// modify it under the terms of the GNU Lesser General Public
+/// License as published by the Free Software Foundation; either
+/// version 3 of the License, or (at your option) any later version.
+///
+/// This program is distributed in the hope that it will be useful,
+/// but WITHOUT ANY WARRANTY; without even the implied warranty of
+/// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+/// Lesser General Public License for more details.
+///
+/// You should have received a copy of the GNU Lesser General Public
+/// License along with this program; if not, write to the Free Software
+/// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+///
+
+import {Event, EventDispatcher} from "./EventDispatcher";
+export class SceneKeyInteractions {
```

Comment:
I took a look into this and the only thing that could be abstracted is the destroy method, or? As it is dependent on the specific events to describe on, i don't think the other methods could be abstracted as well. So i could do it, but i don't know yet why ;-)